frontend/src/components/dashboard/SideBar.tsx

```diff
@@ -7,9 +7,12 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import SecretVersionList from '@app/ee/components/SecretVersionList';
 
+import { WorkspaceEnv } from '~/pages/dashboard/[id]';
+
```

Comment:
@mv-turtle 
wouldn't be better to move some types to `src/utils/types.ts` for example? the line above will throw an eslint error

Reply:
Yes, I just fixed that. I think that was a bug, it shouldn't have been this way I think